MANIFEST.in

```diff
@@ -12,6 +12,7 @@ include *.sh
 include *.in
 include *.txt
 include .flake8
+include scripts/*
```

Comment:
Better to put:

```
recursive-include scripts *.sh
```

So that we are consistent with the other commands.

Please put this statement also in the alphabetical order near the end.